samples/subsys/ipc/openamp_rsc_table/remote/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.13.1)
+# Copyright (c) 2019  STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+
+project(openamp_rsc_table_remote)
+
+zephyr_compile_definitions(
+  -DMETAL_MAX_DEVICE_REGIONS=2
```

Comment:
What is `-DMETAL_MAX_DEVICE_REGIONS` for?

Reply:
This is used to define the size of the static table that register the memory region. by default value is 1 ( shared memory for rpmsg). Here a second one is used for the resource table. that's why it is redefined